includes/functions/content-sync.php

```diff
@@ -198,7 +198,11 @@ function get_post_data( $post ) {
 			$canonical_url = $yoast_canonical;
 		}
 
-		$keywords = get_post_meta( $post->ID, '_yoast_wpseo_focuskw', true );
+		$yoast_focuskw = get_post_meta( $post->ID, '_yoast_wpseo_focuskw', true );
+		if ( ! empty( $yoast_focuskw ) ) {
```

Comment:
if $yoast_focuskw is empty, then $keywords will be still be an empty string. Do we want to change the initial data type as well?

Reply:
Yes, if it's an empty string please have it as an empty array instead, or `null`

Reply:
We have `array_filter` on $data to remove all empty entries. So, in the case of empty `$yoast_focuskw`, the `keywords` field will be removed from $data and it will not be sent in request. As it is not going to send in a request, I think it's fine to keep it as it is. but it may create confusion for other devs too. So, I updated the initial value to `[]`.
Thanks